### ISSUE TITLE
Add downstream tests

### ISSUE THREAD

**Issue by martindurant** [fsspec/filesystem_spec#1036]
Currently we test against s3fs and gcsfs for any change here, but we should also include others. Help welcome in making workflows for them:
- dask/bytes
- pandas IO (csv? parquet?)
- zarr and/or xarray

**Comment by ianthomas23** [comment]
It looks like PR #1037 addressed some of this as it added a `downstream` CI run that includes `pytest fsspec/tests/test_downstream.py` (testing `pandas` CSV via s3 and `zarr` via s3) and also the downstream `dask/dask/bytes` tests. Is that all that is required at the moment or is more needed?

**Comment by martindurant** [comment]
Right, let's close this. We can always do more, but this is OK for now.
